lib/services/firestore_service.dart

```diff
@@ -43,6 +43,7 @@ class FirebaseService {
         "rating": rating,
         "type": type,
         "facilities": facilities,
+        "registeredBy": data["registeredBy"],
```

Comment:
ここは、38行目に
```
 final Map<String, dynamic> registeredBy = data['registeredBy'] ?? {};
 ```
 をのように追加し、

```suggestion
       "registeredBy": registeredBy,
```
として、統一しましょうか。

Reply:
了解です！